docbook/Admin_Guide/en-US/config/issues.xml

```diff
@@ -0,0 +1,119 @@
+%BOOK_ENTITIES;
+]>
+<section id="admin.config.issues" lang="en-US">
+	<title>Issues visibility</title>
+	 <para>
+		By default, all issues are visible to any user within a project. To limit the visibility of issues there are several mechanisms.
+	</para>
+	 <section id="admin.config.issues.private">
+		<title>Public/Private view status</title>
+		 <para>
+			A view status flag can be set, for an issue, to be either public or private. Private issues are accesible by the user who created it, and by those users that meet a threshold defined in <literal>$g_private_bug_threshold</literal>.
+		</para>
+		 <para>
+			Refer to the following configuration options related to issue view status configurations:
+		</para>
+		 <variablelist>
+			<varlistentry>
+				<term>$g_default_bug_view_status</term>
+				 <listitem>
+					<para>
+						The default viewing status for the new bug (VS_PUBLIC or VS_PRIVATE). The default is VS_PUBLIC.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_private_bug_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to view any private issue within a project.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_set_view_status_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to set an issue to Private/Public.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_change_view_status_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to update the view status while updating an issue.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+
+		</variablelist>
+
+	</section>
+	 <section id="admin.config.issues.limitreporters">
+		<title>"Limit reporters" configuration (deprecated)</title>
+		 <para>
+			When the option <literal>$g_limit_reporters</literal> is enabled, users that are reporters in a project, or lower access level, can only see the issues they reported. Any issue reporter by other users are not accessibel by them.
+		</para>
+		 <para>
+			The <literal>$g_limit_reporters</literal> option is supported only for ALL_PROJECTS, this means that it's a global setting that affects all projects.
+		</para>
+		 <para>
+			Note that the definition of <emphasis>reporter</emphasis> in this context is the actual access level for which a user is able to report issues, and is determined by <literal>$g_report_bug_threshold</literal>. Additionally, that threshold can have different values in each project. Being dependant of that threshold, the behaviour of this option is not well defined when the reporting threshold is configured as discrete values with gaps, instead of a simple threshold. In that scenario, the visibilty is determined upon the minimum access level contained in the <literal>$g_report_bug_threshold</literal> access levels array.
+		</para>
+		 <note>
```

Comment:
Maybe this should be `<warning>` instead of `<note>` ?

Since this is deprecated, consider moving the section down, after new _Limited view configuration_ section.

Reply:
That paragraph is not an actual `<note>` block. As there's already another note block about the deprecation of this option, i though it was cleaner this way.
If you consider this should be a real, warning block, i'm ok, let me know.

Reply:
I guess it's fine like that.

Reply:
I still think the deprecated section should be moved down though.